libs/core/src/lib/resizable-card-layout/resizable-card-layout.component.scss

```diff
@@ -0,0 +1,15 @@
+@import '~fundamental-styles/dist/resizable-card-layout';
+
+.fd-resizable-card-layout {
+    position: relative;
+}
+
+.fd-resizable-card-layout--lg {
+    padding-left: 0.5rem !important;
+    padding-right: 0.5rem !important;
+}
+
+.fd-resizable-card-layout--xl {
+    padding-left: 2.5rem !important;
+    padding-right: 2.5rem !important;
+}
```

Comment:
Let's not use important anywhere, where it's not needed. You can put stronger selector

Reply:
i was over-riding values coming from fundamental-styles.
but now using "resizable-card-layout.css" directly. this can be removed once styles are released and version upgraded.